pandas/tests/indexing/common.py

```diff
@@ -1,6 +1,6 @@
 """ common utilities """
-
 import itertools
+from typing import Dict, Hashable, Union
```

Comment:
@topper-123 we have `ignore_errors=True` for `pandas.tests.*` in `setup.cfg`

so I am -1 on adding types that aren't checked. If not correct could be more confusing.

I am not against adding types to the tests, but we need to enable the reporting of errors first IMO.

Reply:
I didn't know that, makes sense that type checking should be turned on before we use type hints. I'll remove the type hints.

I'm also +1 for adding type checking on for tests; they do make it easier to understand what you're looking at.